packages/cli/src/commands/account/claim-keybase.ts

```diff
@@ -96,21 +92,26 @@ export default class ClaimKeybase extends ClaimCommand {
       ) {
         await this.attemptAutomaticProofUpload(claim, signature, username, address)
       } else {
-        this.printManualInstruction(signedClaim, username, address)
+        this.printManualInstruction(claim, signature, username, address)
       }
     } catch (error) {
       cli.action.stop('Error')
       console.error(
         'Could not automatically finish the proving, please complete this step manually.\n\n ' +
           error
       )
-      this.printManualInstruction(signedClaim, username, address)
+      this.printManualInstruction(claim, signature, username, address)
     }
   }
 
-  printManualInstruction(claim: SignedClaim, username: string, address: string) {
+  printManualInstruction(
+    claim: KeybaseClaim,
+    signature: string,
+    username: string,
+    address: string
+  ) {
     const fileName = proofFileName(address)
-    writeFileSync(fileName, JSON.stringify(claim))
+    writeFileSync(fileName, JSON.stringify({ claim, signature }))
```

Comment:
what happens if this file already exists?

Reply:
We'll overwrite it